### PR TITLE
Avoid chown if the file already has correct owner and group

### PR DESCRIPTION
**Pull request by secfree**
### What changes are proposed in this pull request?

Avoid chown if the file already has correct owner and group

### Why are the changes needed?

- The client may send a request to change owner&group even the file already has correct values
- The request may be rejected because "chown" needs supergroup privilege

### Does this PR introduce any user facing changes?

No

### CODE REVIEW

File: integration/fuse/src/test/java/alluxio/fuse/auth/AbstractAuthPolicyTest.java [hunk 1]
```diff
@@ -81,6 +81,11 @@ public void setUserGroup() throws Exception {
     URIStatus status = mFileSystem.getStatus(uri);
     Assert.assertEquals(userName, status.getOwner());
     Assert.assertEquals(groupName, status.getGroup());
+
+    // do not call "setAttribute" if the file already has correct owner and group
```
Comment (jiacheliu3) [inline integration/fuse/src/test/java/alluxio/fuse/auth/AbstractAuthPolicyTest.java:85]: Do you intend to verify the `mFileSystem.setAttribute()` is not called?

**Review by jiacheliu3 — CHANGES_REQUESTED** [review]
Mostly LGTM with a minor comment, thanks for the improvement!

File: integration/fuse/src/test/java/alluxio/fuse/auth/AbstractAuthPolicyTest.java (hunk 1 above)
Comment (secfree) [reply integration/fuse/src/test/java/alluxio/fuse/auth/AbstractAuthPolicyTest.java]: Yes. I just adjusted the comment, please help review once more. Thank you.

Comment (secfree) [reply integration/fuse/src/test/java/alluxio/fuse/auth/AbstractAuthPolicyTest.java]: It's not easy to do mock/spy here, so I do it in this way.

**Review by jiacheliu3 — APPROVED** [review]
LGTM, thanks!

**Comment by jiacheliu3** [comment]
/ping @LuQQiu Could you take a quick look? Thanks!

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks for the fix!

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Pull request merged**